debezium-embedded/src/test/java/io/debezium/transforms/AbstractVectorToJsonConverterTest.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceConnector;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.debezium.config.Configuration;
+import io.debezium.data.Envelope;
+import io.debezium.data.Json;
+import io.debezium.data.VerifyRecord;
+import io.debezium.doc.FixFor;
+import io.debezium.embedded.async.AbstractAsyncEngineConnectorTest;
+import io.debezium.jdbc.JdbcConnection;
+
+/**
+ * Abstract base class for testing the {@link VectorToJsonConverter} transformation.
+ *
+ * @author Chris Cranford
+ */
+public abstract class AbstractVectorToJsonConverterTest<T extends SourceConnector> extends AbstractAsyncEngineConnectorTest {
```

Comment:
Maybe we just need a limited IT for this SMT and most of the functionality can be tested via unit test?

Reply:
Sure, I assume by limited IT you'd like me to pick a connector and roll a simple example through that connector to verify that it does what it should, i.e. PostgreSQL?